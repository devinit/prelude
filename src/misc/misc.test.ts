import 'jest';
import {getShortURL, sendEmail} from '.';

describe('misc tests', () => {
    it('should return short url of a long url', async () => {
        const url =
          await getShortURL('http://212.71.254.23:9999/country/uganda?state={"year":2015,"budgetType":"actual"}');
        expect(url).toMatchSnapshot();
      },
      20000,
    );
    it.skip('should run without errors while sending email', async () => {
      const response = await sendEmail({
        message: 'test',
        token: 'e2DQks99XapU6w2s1',
        emails: ['epicallan.al@gmail.com'],
        subject: 'test email from datahub'
      });
      expect(response.status).toBe(200);
    });
});
