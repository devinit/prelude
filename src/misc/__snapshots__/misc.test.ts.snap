// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`misc tests should return short url of a long url 1`] = `"http://bit.ly/2wamIhE"`;
